### PR TITLE
replace travis (part 2): run tests in github actions

### PR DESCRIPTION
**Pull request by aabmass**
- Filled in the github workflow to run tox on supported python versions, based largely on the [OpenTelemetry workflow](https://github.com/open-telemetry/opentelemetry-python/blob/6411755abd078a3eecabd88931f7beb5876ea2e9/.github/workflows/test.yml).
- Deleted `.travis.yml`

### CODE REVIEW

File: contrib/opencensus-ext-azure/tests/test_azure_heartbeat_metrics.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def test_heartbeat_metric_get_metric_init(self):
         self.assertEqual(metric.NAME, 'Heartbeat')
         keys = list(metric.properties.keys())
         values = list(metric.properties.values())
-        self.assertEqual(len(keys), 2)
+        self.assertEqual(len(keys), 5)
```
Comment (aabmass) [inline contrib/opencensus-ext-azure/tests/test_azure_heartbeat_metrics.py:100]: @lzchen this test was failing on all python versions. Change looks ok?

Comment (lzchen) [reply contrib/opencensus-ext-azure/tests/test_azure_heartbeat_metrics.py:100]: On your local? It's passing for me on local.

Comment (aabmass) [reply contrib/opencensus-ext-azure/tests/test_azure_heartbeat_metrics.py:100]: No, just on the action run. I'll change it back so you can see hold on

Comment (aabmass) [reply contrib/opencensus-ext-azure/tests/test_azure_heartbeat_metrics.py]: https://github.com/census-instrumentation/opencensus-python/pull/980/checks?check_run_id=1501556499#step:6:45

**Review by lzchen — APPROVED** [review]

**Pull request merged**